docs/src/app/components/pages/components/DatePicker/Page.jsx

```diff
@@ -59,6 +62,13 @@ const DatePickerPage = () => (
       <DatePickerExampleControlled/>
     </CodeExample>
     <CodeExample
+      title="Disabled dates example"
+      description={descriptions.disabledDates}
+      code={datePickerExampleDisableDatesCode}
+    >
+      <DatePickerExampleDisableDates/>
```

Comment:
`<DatePickerExampleDisableDates />`

Could you also fix the others? Not sure why this isn't getting linted...